sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -74,10 +74,10 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
         /// </summary>
         /// <param name="endpoint">A <see cref="Uri"/> to the service the client
         /// sends requests to.  Endpoint can be found in the Azure portal.</param>
-        /// <param name="subscriptionKey">The subscription key used to access
+        /// <param name="credentials">The subscription key used to access
         /// the service.</param>
-        public TextAnalyticsClient(Uri endpoint, string subscriptionKey)
-            : this(endpoint, subscriptionKey, new TextAnalyticsClientOptions())
+        public TextAnalyticsClient(Uri endpoint, ServiceSubscriptionKey credentials)
```

Comment:
Let's also add the overload that takes a string subscriptionKey and instantiates a TextAnalyticsSubscriptionKeyCredential from it.  The arch board can tell us if they don't like it in our reviews with them, but the service team has requested that we make this as simple as possible for customers.  We'll want docstrings that make it clear that if you use that constructor, you won't be able to rotate credentials.

Reply:
I just saw that Krista and Johan decided not to add this for Python.  Let's understand why, and if their thinking applies here, let's follow suit.